pkg/skaffold/deploy/labels.go

```diff
@@ -269,46 +116,81 @@ func labelDeployResults(labels map[string]string, results []Artifact) {
 	}
 }
 
-func addSkaffoldLabels(labels map[string]string, m *metav1.ObjectMeta) {
-	if m.Labels == nil {
-		m.Labels = map[string]string{}
+func addLabels(labels map[string]string, accessor metav1.Object) {
+	for k, v := range constants.Labels.DefaultLabels {
```

Comment:
Shouldn't we apply `DefaultLabels` to  objLabels first and then apply `labels`?

Reply:
fixed